apps/profiles/models.py

```diff
@@ -59,6 +59,24 @@ class Meta:
     def __str__(self):
         return "{0}:{1} - {2}".format(self.user.username, self.student_id, self.klass.title)
 
+    @property
+    def klass_grade(self):
+        grade_quantity = 0
+        grade_total = 0
+        for definition in self.klass.homework_definitions.all():
+            submission = self.submitted_homeworks.filter(definition=definition).order_by('created').last()
+            if submission:
+                grade_object = submission.grades.last()
```

Comment:
could we do this in 1 query like how we get the latest homework for that table?

if we do `whatever.klass_grade` in a template or something this will spin up a few queries we maybe don't want

Reply:
Tried prefetching these, but get an increase in the query count. Saving for later if we have delay issues when this is called.

Reply:
cool